### PR TITLE
[enhancement](config) add some pre-process and pre-check for BE storage config attentions in docs

### PR DESCRIPTION
**Pull request by Hastyshell**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by TangSiyang2001** [comment]
run buildall

File: be/src/olap/options.cpp [hunk 1]
```diff
@@ -156,11 +166,17 @@ Status parse_conf_store_paths(const string& config_path, std::vector<StorePath>*
...
+        }
     }
-    if (paths->empty() || (path_vec.size() != paths->size() && !config::ignore_broken_disk)) {
+    if ((path_vec.size() != paths->size() && !config::ignore_broken_disk)) {
         return Status::Error<INVALID_ARGUMENT>("fail to parse storage_root_path config. value={}",
                                                config_path);
     }
+    if (hdd_cnt == 0) {
+        return Status::Error<INVALID_ARGUMENT>("should a least have one hdd path");
+    }
```
Comment (dataroaring) [inline be/src/olap/options.cpp:179]: We should remove this check.

**Review by dataroaring — CHANGES_REQUESTED** [review]
Add some ut?

**Comment by hello-stephen** [comment]
(From new machine)TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 47.93 seconds
 stream load tsv:          513 seconds loaded 74807831229 Bytes, about 139 MB/s
 stream load json:         21 seconds loaded 2358488459 Bytes, about 107 MB/s
 stream load orc:          64 seconds loaded 1101869774 Bytes, about 16 MB/s
 stream load parquet:          32 seconds loaded 861443392 Bytes, about 25 MB/s
 insert into select:          29.8 seconds inserted 10000000 Rows, about 335K ops/s
 storage size: 17162210193 Bytes

**Comment by dataroaring** [comment]
run buildall

**Comment by hello-stephen** [comment]
(From new machine)TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 48.27 seconds
 stream load tsv:          515 seconds loaded 74807831229 Bytes, about 138 MB/s
 stream load json:         20 seconds loaded 2358488459 Bytes, about 112 MB/s
 stream load orc:          65 seconds loaded 1101869774 Bytes, about 16 MB/s
 stream load parquet:          31 seconds loaded 861443392 Bytes, about 26 MB/s
 insert into select:          29.4 seconds inserted 10000000 Rows, about 340K ops/s
 storage size: 17162103400 Bytes

**Comment by hello-stephen** [comment]
(From new machine)TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 47.47 seconds
 stream load tsv:          513 seconds loaded 74807831229 Bytes, about 139 MB/s
 stream load json:         20 seconds loaded 2358488459 Bytes, about 112 MB/s
 stream load orc:          64 seconds loaded 1101869774 Bytes, about 16 MB/s
 stream load parquet:          31 seconds loaded 861443392 Bytes, about 26 MB/s
 insert into select:          29.2 seconds inserted 10000000 Rows, about 342K ops/s
 storage size: 17162210923 Bytes

**Comment by TangSiyang2001** [comment]
run buildall

**Comment by TangSiyang2001** [comment]
run clickbench

**Comment by hello-stephen** [comment]
(From new machine)TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 48.73 seconds
 stream load tsv:          511 seconds loaded 74807831229 Bytes, about 139 MB/s
 stream load json:         20 seconds loaded 2358488459 Bytes, about 112 MB/s
 stream load orc:          64 seconds loaded 1101869774 Bytes, about 16 MB/s
 stream load parquet:          30 seconds loaded 861443392 Bytes, about 27 MB/s
 insert into select:          29.2 seconds inserted 10000000 Rows, about 342K ops/s
 storage size: 17162161622 Bytes

File: be/src/olap/options.cpp [hunk 2]
```diff
@@ -148,6 +148,14 @@ Status parse_root_path(const string& root_path, StorePath* path) {
 
 Status parse_conf_store_paths(const string& config_path, std::vector<StorePath>* paths) {
     std::vector<string> path_vec = strings::Split(config_path, ";", strings::SkipWhitespace());
+    if (path_vec.empty()) {
+        return Status::Error<INVALID_ARGUMENT>(
+                "parse storage_root_path config result empty. value={}", config_path);
```
Comment (dataroaring) [inline be/src/olap/options.cpp:153]: Compute node may be broken.

**Review by dataroaring — CHANGES_REQUESTED** [review]

**Comment by TangSiyang2001** [comment]
run buildall

**Comment by TangSiyang2001** [comment]
run buildall

**Review by dataroaring — APPROVED** [review]
LGTM

**Review by freemandealer — APPROVED** [review]
LGTM

**Comment by hello-stephen** [comment]
(From new machine)TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 48.26 seconds
 stream load tsv:          514 seconds loaded 74807831229 Bytes, about 138 MB/s
 stream load json:         20 seconds loaded 2358488459 Bytes, about 112 MB/s
 stream load orc:          65 seconds loaded 1101869774 Bytes, about 16 MB/s
 stream load parquet:          32 seconds loaded 861443392 Bytes, about 25 MB/s
 insert into select:          29.4 seconds inserted 10000000 Rows, about 340K ops/s
 storage size: 17162576938 Bytes

**Comment by hello-stephen** [comment]
(From new machine)TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 48.28 seconds
 stream load tsv:          512 seconds loaded 74807831229 Bytes, about 139 MB/s
 stream load json:         20 seconds loaded 2358488459 Bytes, about 112 MB/s
 stream load orc:          65 seconds loaded 1101869774 Bytes, about 16 MB/s
 stream load parquet:          31 seconds loaded 861443392 Bytes, about 26 MB/s
 insert into select:          29.1 seconds inserted 10000000 Rows, about 343K ops/s
 storage size: 17162480628 Bytes

**Pull request merged**